great_ape_safe/ape_api/univ3.py

```diff
@@ -0,0 +1,50 @@
+import json
+import os
+from brownie import interface, Contract
+from helpers.addresses import registry
+
+
+class UniV3:
+    def __init__(self, safe):
+        self.safe = safe
+
+        # contracts
+        self.NonfungiblePositionManager = safe.contract(
+            registry.eth.uniswap.NonfungiblePositionManager
+        )
```

Comment:
maybe introduce interface file for this?

Reply:
included the interface, from now on when creating the classes we are leaning towards always adding the interface?

Reply:
yea i think this is more robust and will prevent these 'invisible' brownie errors where a contract is loaded with the wrong interface. also i like researching the contracts and realising a bit better what is what.